documents/validator.go

```diff
@@ -400,6 +400,44 @@ func anchorRepoAddressValidator(anchoredRepoAddr common.Address) Validator {
 	})
 }
 
+// attributeValidator validates the signed attributes.
+func attributeValidator(repo anchors.AnchorRepository, idSrv identity.Service) Validator {
+	return ValidatorFunc(func(_, model Model) (err error) {
+		ts, err := model.Timestamp()
+		if err != nil {
+			return err
+		}
+
+		attrs := model.GetAttributes()
```

Comment:
Would it be interesting to have a getter in the model for all SignedAttributes?

Reply:
The idea to have a single function to give all attributes and keep the model API simple